src/braket/aws/aws_session.py

```diff
@@ -235,6 +235,12 @@ def create_quantum_task(self, **boto3_kwargs) -> str:
         Returns:
             str: The ARN of the quantum task.
         """
+        # Add reservation arn if available and device is correct.
+        device_arn = os.getenv("AMZN_BRAKET_DEVICE_ARN_TEMP")
+        reservation_arn = os.getenv("AMZN_BRAKET_RESERVATION_ARN_TEMP")
```

Comment:
Probably don't need `_TEMP`

Reply:
ok I will remove it

Reply:
actually, `"AMZN_BRAKET_DEVICE_ARN"` might need the prefix to not conflict with the env var used in hybrid jobs.